ci/docker/runtime_functions.sh

```diff
@@ -33,6 +33,18 @@ clean_repo() {
     git submodule update --init --recursive
 }
 
+check_cython() {
+    set -ex
+    local python_ver=$1
+    if [ "$(echo -e 'import mxnet as mx\nprint(mx.nd._internal.NDArrayBase.__module__)' | python${python_ver})" != "mxnet._cy${python_ver}.ndarray" ]; then
```

Comment:
can we use python -c and move it before the conditional?

Reply:
I couldn't figure out how to run multiline code with `python -c`. Anyway this kind of piping is a standard unix way. Any reason to prefer `python -c`?

Reply:
I guess doesn't make a difference. Can we store in a variable with propper name? It's difficult to read inside the if condition.